summit/amor01/shutdown_love

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /deploy-lsstts/.love-cred.sh
```

Comment:
You may want to stick to a common naming convention. Here the `.love_secrets` file does have an extension but in the lsst-teststand-ts1 script it doesn't.

Reply:
OK. I'll make the file the same between the sites and adjust the scripts accordingly.